README.md

```diff
@@ -1,37 +1,79 @@
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
 
-### StarterCmd
+<img width="500" alt="Screenshot CageChooser" src="https://user-images.githubusercontent.com/1786772/43733677-c50f6b72-99b5-11e8-9831-69556fd33246.png">
 
-A user interface for starting the `CageManager` and an application in it. Directly communicating with the `CageService`.
+</details>
 
-### ImageSelectDialog
+### CageConfigurator
 
-A user interface to select an icon to be displayed on the user's desktop. `config.txt` will be created in `C:\sharkcage` with a line containing `picture: ` followed by the index of the icon. The `CageService` reads this configuration file and searches for that line and opens the dialog if the file contains no such line.
+The `CageConfigurator` is a user interface to create a config file including an icon to be displayed on the secure desktop, the program which should be started and an optionally additional program. The additional program can be choosen out of a list of "trustworthy" applications.
+ As soon as the config file has been saved, a link to the config which is stored under `C:\Users\Public\Documents\SharkCage\` will be saved in the Registry under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`.
+ The config contains json data and is protected using ACL's to be read and edited only by the `CageConfigurator` in a secure environment, the `CageService` as well as the administrator group.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+<img width="1680" alt="Screenshot CageConfigurator" src="https://user-images.githubusercontent.com/1786772/43733731-e88c2e50-99b5-11e8-84a5-860351997f55.png">
+
+</details>
+
+### SharkCageInstaller
+
+The `SharkCageInstaller` is used to install all project applications, starts the `CageService` and sets some keys in the registry.
 
 ## Installation
 
-1. Build the project
-1. Create the folder `C:\sharkcage` and copy `CageService.exe`, `CageManager.exe` and `ImageSelectDialog.exe` to this folder.
-1. Open `cmd` or `powershell` as administrator and add `CageService.exe` to the services: `sc create "shark-cage-service" binPath="C:\sharkcage\CageService.exe" obj= "NT AUTHORITY\SYSTEM"`
-1. Configure and start the `CageService`: `sc config "shark-cage-service" start=auto | sc start "shark-cage-service"`
+1. The `SharkCageInstaller` is used, to install all programs for this project (`CageService`, `CageManager`, `CageChooser` and `CageConfigurator`) and is hosted on Github. Follow this link and download the latest version: [Github-Releases](https://github.com/SharkCagey/HTWG_shark_cage/releases).
+1. Out of security perspective before installing, please make sure that the `SharkCageInstaller` is signed using the certificate issued to the HTWG Konstanz with the following fingerprint: `ADBE74BD39789DD111815DE59C60D715143E4620`.
```

Comment:
Please make sure the `SharkCageInstaller` is signed using the certificate issued to the HTWG Konstanz with the following fingerprint: `ADBE74BD39789DD111815DE59C60D715143E4620` to avoid any unnecessary security risks.